apps/ocsp.c

```diff
@@ -1225,6 +1244,7 @@ static void make_ocsp_response(BIO *err, OCSP_RESPONSE **resp, OCSP_REQUEST *req
             ASN1_TIME_free(revtm);
             ASN1_GENERALIZEDTIME_free(invtm);
         }
+        OCSP_CERTID_free(cid_resp_md);
```

Comment:
It would be a no-op to do so at present, but I would consider NULL-ing out ```cid``` here.

Reply:
True, there is also a way tweak the code slightly to avoid aliases